packages/pglite/src/extensionUtils.ts

```diff
@@ -99,7 +99,15 @@ function loadExtension(
           false,
         )
       } else {
-        mod.FS.writeFile(filePath, file.data)
+        try {
```

Comment:
Needed to create full folder path before writing file. Is there an emscripten FS API call to create the full path if not exists?